src/components/notification/notification.scss

```diff
@@ -157,6 +175,10 @@
           ) / 2
       );
     }
+
+    :host([size='s']) & {
+      margin-block-start: 0;
```

Comment:
Would it make sense to introduce a css var as well ?. and is it still working without and with title?

Reply:
Not sure it improved much in terms of readability though :/

Reply:
it's the same I think :D but a little more like our standards. for me ok. I hope you can live with it as well :D